Classes/Event/GlossaryKeyEvent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WebVision\Deepltranslate\Core\Event;
+
+use WebVision\Deepltranslate\Core\Domain\Dto\CurrentPage;
+
+final class GlossaryKeyEvent
```

Comment:
Please add a class php-docblock comment describing when/where this event is dispatched and what it can be used for.